api/rights.proto

```diff
@@ -183,6 +183,10 @@ message Collaborator {
   repeated Right rights = 2 [(validate.rules).repeated.items.enum.defined_only = true];
 }
 
+message GetCollaboratorResponse {
+  repeated Right rights = 2;
```

Comment:
Cleaner to start from `1`

Reply:
Updated the protos with reserved fields and corresponding comments.